### DISCUSSION TITLE
Desktop app states "Incorrect Details" when adding AWS

### DISCUSSION THREAD

**Discussion by erik-siverson** [ente-io/ente#4431] [Q&A]
I'm trying to add an entry into the desktop Ente app for AWS using the secret key but the app just says "Incorrect Details" and directs me to contact support here.  The secret key is 64 alphanumeric characters and the same thing happens both on Windows and the linux AppImage.  That same secret key is loaded correctly into Authenticator Pro on my android phone.  Any suggestions?

**Comment by ua741** [comment]
Can you please share any unused secret key, using which we can reproduce the issue?

**Comment by erik-siverson** [comment]
I'm hesitant to post the key publicly since the account number is embedded.  Is there some way I can send it to you privately?

**Reply by ua741** [reply]
> I'm hesitant to post the key publicly since the account number is embedded. Is there some way I can send it to you privately?

Please write to us support @ente.io and mention this discussion in that email.

**Comment by wrnunzio** [comment]
I am experiencing the same issue with Salesforce 2fa. 
If I use the qr code scan true the mobile app instead of copying and pasting in the desktop app the secret, it works

**Reply by ua741** [reply]
> I am experiencing the same issue with Salesforce 2fa. If I use the qr code scan true the mobile app instead of copying and pasting in the desktop app the secret, it works

Could you please downgrade to an older version on the desktop? There’s a regression in the recent version (4.2.7) that has caused the manual code setup to break. We are working on rolling out a fix.
